blocks/init/src/blocks/custom/featured-posts/components/featured-posts-editor.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import { Fragment } from '@wordpress/element';
+
+export const FeaturedPostsEditor = (props) => {
```

Comment:
spread props like in this example: https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/blocks/custom/card/components/card-editor.js